src/assets/patterns/molecules/artboards/css/artboards.css

```
@@ -76,8 +76,8 @@
 }
 
 .m-artboards__list {
-    flex: 1 0;
-    overflow: auto;
+    flex: 1 1;
```

```suggestion
    flex: auto;
```
The same functionality but is a bit nicer to read.